apps/intellij/src/main/kotlin/dev/nx/console/nxls/WorkspacePathSetting.kt

```diff
@@ -42,6 +42,10 @@ class WorkspacePathSetting(val project: Project) : NxConsoleSettingBase<String?>
     }
 
     override fun getValue(): String? {
+        val value = this.inputField.text
```

Comment:
It can be one line 
`override fun getValue(): String? = inputField.text.ifEmpty { null }`

Reply:
yeah that looks better